<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-BCB52JF" ignored="0" name="Verify title" tests="1" failures="1" timestamp="2021-01-07T15:45:21 IST" time="48.076" errors="0">
  <testcase name="validateTitle" time="0.457" classname="Framework.project.VerifyTitle">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Framework.project.VerifyTitle.validateTitle(VerifyTitle.java:39)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- validateTitle -->
</testsuite> <!-- Verify title -->
